codewof/static/js/question_types/function.js

```diff
@@ -108,3 +114,53 @@ function run_python_code(user_code, test_case) {
         test_case.runtime_error = true;
     }
 }
+
+
+function setTutorialAttributes() {
+    $(".question-text").attr(
+        'data-intro',
+        'This is a description of what the function should do. It will tell you what your function should return (or print) in order to pass the tests.\
+        Make sure to pay close attention to whether your function should return or print a value!'
+    );
+    $("#python-editor").attr(
+        'data-intro',
+        "This is where you enter your code to solve the problem."
+    );
+    $("#run_code").attr(
+        'data-intro',
+        "Click 'Run code' to test your function against our test cases and see if they pass!"
+    );
+    $("#test-case-table").attr(
+        'data-intro',
+        "These are the test cases that will be run against your function."
+    );
+    // the first row in the test case table
+    $('#test-case-table tbody tr:nth-child(1)').attr(
+        'data-intro',
+        'Here is the first test case.'
+    );
+    // the input for the first test case
+    $('#test-case-table tbody tr:nth-child(1) td:eq(0)').attr(
+        'data-intro',
+        'This is how your function will be called. Pay close attention to the input that is passed to the function.'
```

Comment:
It's given the term 'test code' in the debugging helper so maybe be consistent here

"This is the test code that is run for this particular test. Pay close ..."